apps/docs/src/content/reference/extras/csm.mdx

```diff
@@ -0,0 +1,123 @@
+| lightIntensity       | number                                                                                 | Same as `THREE.DirectionalLight.intensity`.                                                                                                                                                                                                                                                                                                                                                                      |
+| lightNear            | number                                                                                 | Shadow camera frustum near plane                                                                                                                                                                                                                                                                                                                                                                                 |
+| lightFar             | number                                                                                 | Shadow camera frustum far plane                                                                                                                                                                                                                                                                                                                                                                                  |
+| lightMargin          | number                                                                                 | Defines how far shadow camera is moved along z axis in cascade frustum space. The larger is the value the more space `LightShadow` will be able to cover. Should be set to high values for scenes with huge objects.                                                                                                                                                                                             |
+| customSplitsCallback | `(cascades: number, cameraNear: number, cameraFar: number, breaks: number[]) => void;` | A callback to compute custom cascade splits when mode is set to `"custom"`. Callback should accept three number parameters: `cascadeCount`, `nearDistance`, `farDistance` and return an array of split distances ranging from 0 to 1, where 0 is equal to `nearDistance` and 1 is equal to `farDistance`.                                                                                                        |
+
+For the original implementation, refer to [this github repository](https://github.com/StrandedKitty/three-csm).
```

Comment:
This is confusing, since the implementation is slightly different. 

```suggestion
For the original implementation, refer to [the Three.js GitHub repository](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/csm/CSM.js).
```